docs/static/ls-ls-config.asciidoc

```diff
@@ -0,0 +1,111 @@
+[[ls-to-ls]]
+=== Configuring Logstash-to-Logstash Communication
+
+You can set up communication between two Logstash machines by connecting the Lumberjack output to the Beats input. You may need this configuration if the data path crosses network or firewall boundaries, for example. If you don't have a compelling need for Logstash-to-Logstash communication, then don't implement it.
+
+[[ls-to-ls-overview]]
+==== Configuration overview
+
+Use the Lumberjack protocol to connect two Logstash machines.
+
+. Generate a trusted SSL certificate (required by the lumberjack protocol).
+. Save the SSL certificate on the upstream Logstash machine.
```

Comment:
"Save" is a weird verb for server side software, since there isn't really a 'save' action in the software operators use for the most part, and there's no unsaved state.

I think "copy" or "put" make more sense in general for Logstash and similar software.